Riot/Modules/TabBar/MasterTabBarController.m

```diff
@@ -405,6 +405,15 @@ - (void)removeMatrixSession:(MXSession *)mxSession
 
 - (void)onMatrixSessionStateDidChange:(NSNotification *)notif
 {
+    MXSession *session = (MXSession*)notif.object;
+    if (session.state == MXSessionStateStoreDataReady)
+    {
+        if (session.crypto.crossSigning && session.crypto.crossSigning.state == MXCrossSigningStateCrossSigningExists)
```

Comment:
`LegacyAppDelegate.handleAppState()` would be a better place for this.

Reply:
Code moved